docker/nvidia/Dockerfile_nvidia

```diff
@@ -1,6 +1,6 @@
-FROM dalg24/dtk-stack:latest
+FROM @BASE_IMAGE@
```

Comment:
Look in `docker/nvidia/setup_nvidia_docker_compose.py`.
I replace it with image name of the service that you extend.